docs/howto/partitions.rst

```diff
@@ -64,26 +70,31 @@ To add partition support to an application, two basic changes are needed:
       but partitions cannot be enabled via :class:`AppFeatures
       <craft_application.AppFeatures>`!
 
-#. Define the list of partitions
+   .. NOTE::
+      In unit tests, you may need to call this between tests, otherwise the
+      :class:`Features <craft_parts.Features>` global singleton may complain as
+      successive tests try to enable partitions repeatedly.
```

Comment:
```suggestion
    .. Tip::
      In unit tests, the :class:`Features <craft_parts.Features>` global
      singleton may raise exceptions when successive tests repeatedly try to
      enable partitions.

      To prevent these excess errors, reset the features at the start of each
      test:
```